### ISSUE TITLE
 Request: as_coco Parameter Adjustment for sv.DetectionDataset in Supervision API

### ISSUE THREAD

**Issue by YoungjaeDev** [roboflow/supervision#1181]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Description

I am currently utilizing the `as_coco` function from **_sv.DetectionDataset_** within the Supervision API extensively and find it incredibly useful. However, I've encountered an issue when importing these labels into CVAT label tool. It seems that CVAT accepts **category_id starting from 1**, whereas the as_coco output starts with category_id at 0. This discrepancy leads to import failures.

Would it be possible to add a parameter to the as_coco function to adjust the starting index of category_id? This feature would greatly enhance compatibility with CVAT and streamline the workflow for users facing similar issues.

Thank you for considering this enhancement. I look forward to any updates you might have on this request.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SkalskiP** [comment]
Hi @YoungjaeDev 👋🏻 does CVAT requires only `category_id` to start from 1 or is it also `image_id`?

**Comment by YoungjaeDev** [comment]
@SkalskiP 
Just category_id. Image_id doesn't matter for importing data

**Comment by YoungjaeDev** [comment]
@SkalskiP 

cvat.apps.dataset_manager.bindings.CvatImportError: Image C00_012_0007_000000: can't import annotation #0 (bbox): annotation has no label.

This is error log

**Comment by SkalskiP** [comment]
I am very willing to implement any necessary changes to facilitate your use of supervision in combination with CVAT. I just need to know if changing the`image_id` numbering to start from `1` is the only requirement you have.

**Comment by YoungjaeDev** [comment]
@SkalskiP 

No, the `image_id` starts with 1, which is a CVAT requirement, and Supervision does that. However, CVAT requires the **_category_id_** to start with 1, and Supervision starts with 0.

**Comment by SkalskiP** [comment]
Okey! 🙏🏻

**Comment by YoungjaeDev** [comment]
Yes, thank you
